tgui/packages/tgui/interfaces/AirAlarm.js

```diff
@@ -237,13 +237,13 @@ const AirAlarmVentsView = (props, context) => {
           </LabeledList.Item>
           <LabeledList.Item label="Pressure Checks">
             <Button
-              content="Internal"
+              content="External"
```

Comment:
Please keep `Internal` and `External` on the left and right respectively and update the check logic itself. Also remember to recompile TGUI and test your code instead of just submitting.